src/components/USBController.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
```

Comment:
nit: We should call this `USBControllerButton` to make obvious that this component returns a button.

Reply:
That's fair. In the interest of time, and since this should be standardized in `election-manager` too, I'm going to make a followup ticket: https://github.com/votingworks/vxmail/issues/110